domain/src/main/java/com/anytypeio/anytype/domain/types/CreateObjectType.kt

```diff
@@ -1,17 +1,16 @@
 package com.anytypeio.anytype.domain.types
 
 import com.anytypeio.anytype.core_models.Id
-import com.anytypeio.anytype.core_models.ObjectWrapper
+import com.anytypeio.anytype.core_models.Struct
 import com.anytypeio.anytype.domain.base.AppCoroutineDispatchers
 import com.anytypeio.anytype.domain.base.ResultInteractor
 import com.anytypeio.anytype.domain.block.repo.BlockRepository
 
-// TODO rename to "CreateObjectType" after refactoring
-class CreateType(
+class CreateObjectType(
     private val repo: BlockRepository,
     dispatchers: AppCoroutineDispatchers
-) : ResultInteractor<CreateType.Params, ObjectWrapper.Type>(dispatchers.io) {
-    override suspend fun doWork(params: Params): ObjectWrapper.Type {
+) : ResultInteractor<CreateObjectType.Params, Struct?>(dispatchers.io) {
```

Comment:
on `domain` level, i would prefer to get the result already wrapped in the `ObjectWrapper.Type`. what do you think?

Reply:
changed to ObjectWrapper.Type